resources/views/layouts/admin.blade.php

```diff
@@ -33,23 +34,40 @@
                 </li>
             </ul>
         </div>
+        <!-- Sidebar -->
+
         <div class="col-md-10 d-flex flex-column">
-            <nav class="p-3 shadow text-end">
-                <span class="profile-font">{{ __('admin/Layout.profile_name') }}</span>
-                <img class="img-profile rounded-circle" src="{{ asset('/img/user.png') }}" style="width: 40px; height: 40px;">
-            </nav>
-            <div class="g-0 m-5 flex-grow-1">
-                @yield('content')
-            </div>
-        </div>
+    <!-- Admin Navbar -->
+    <nav class="p-3 shadow text-end">
+    @auth
+        <a href="{{ route('user.show', ['id' => Auth::user()->id]) }}" class="profile-font text-decoration-none text-light">
+            <span>{{ Auth::user()->name }}</span>
+            @if(Auth::user()->image) <!-- Asegurarse de que haya una imagen -->
```

Comment:
Remove this unnecessary comment. @Juanstevan1

Reply:
done